tests/test_batch_select.ts

```diff
@@ -0,0 +1,112 @@
+import { Wrapper } from '@vue/test-utils';
+
+import _ from 'lodash';
+
+import BatchSelect from "@/components/batch_select.vue";
+
+import { managed_mount } from '@/tests/setup';
+import { emitted } from '@/tests/utils';
+
+describe('Field binding tests', () => {
```

Comment:
Could you add a couple tests that cover:
- Opening and closing the modal (clicking the "Cancel" button and making the modal component emit a "close" event; I can point you towards some similar tests that do this if you like)
- The `value` array changing from outside (to test the watcher)